openssl.yaml

```diff
@@ -38,6 +38,10 @@ pipeline:
       uri: https://www.openssl.org/source/openssl-${{package.version}}.tar.gz
       expected-sha256: a0ce69b8b97ea6a35b96875235aa453b966ba3cba8af2de23657d8b6767d6539
 
+  - uses: patch
+    with:
+      patches: CVE-2023-4807.patch
```

Comment:
Are you sure this is the right CVE ID?

Reply:
Ah, I see the link now, thanks!

Reply:
Yea, another case of NVD missing the entry...